docs/angular/guides/storybook/storybook-composition.md

```diff
@@ -0,0 +1,174 @@
+
+## How to use it
+
+All you need is a URL of a live Storybook, and a "host" Storybook. In the `.storybook/main.js` file of the "host" Storybook, inside `module.exports` you add a new `refs` attribute, which will contain the link(s) for the composed Storybook(s).
+
+In the example below, we have a host Storybook running on local port 4400 (http://localhost:4400) - not displayed here. In it, we want to compose three other Storybooks. The "one-composed" and "two-composed", running on local ports `4401` and `4402` accordingly, as well as the [Storybook website's Storybook](https://next--storybookjs.netlify.app/official-storybook) which is live on the address that you see.
+
+```
+// .storybook/main.js of our Host Storybook - supposably running on port 4400
+
+module.exports = {
+  ...,
+  refs: {
+    'one-composed': {
+      title: 'One composed',
+      url: 'http://localhost:4401',
+    },
+    'two-composed': {
+      title: 'Two composed',
+      url: 'http://localhost:4402',
+    },
+    'storybook-website-storybook': {
+      title: 'The Storybook of the Storybook website',
+      url: 'https://next--storybookjs.netlify.app/official-storybook/',
+    },
+  },
+};
+```
+
+You can always read more in the [official Storybook docs](https://storybook.js.org/docs/angular/workflows/storybook-composition#compose-published-storybooks).
+
+## How to use it in Nx
+
+It's quite easy to use this feature, in Nx and in general, since you do not need to make any code changes, you just need to have the "composed" Storybook instances (the ones you need to "compose") running, choose a "host" Storybook, and just add the composed Storybooks in it's `.storybook/main.js` file.
+
+Nx provides the [`run-many`](https://nx.dev/l/a/cli/run-many) command, which will allow you to easily run multiple Storybooks at the same time. You need to run the `run-many`
+command with the parallel flag set to true `--parallel=true`, because you want to run all your Storybooks in parallel. Since the default amount of parallel activities that you can run
+is set to `3`, you can adjust this command to be of as many Storybooks you want to run in parallel as you need. However, be **very carefull** with putting large numbers in this
+flag, since it can cause big delays or get stuck. You can play around and adjust that number to one your machine runs comfortably with. Keep in mind that you can add in this feature however many live/public Storybooks as you need (Storybooks that you do not run locally).
+
+You can just do:
+
+```
+nx run-many --target=storybook --projects=main-host,one-composed,two-composed,three-composed --parallel=true --maxParallel=4
```

Comment:
I found `--maxParallel` has been  deprecated https://github.com/nrwl/nx/pull/7677

Reply:
Thanks, @puku0x  I'll fix this! :) :)